dotenv.cabal

```diff
@@ -78,6 +78,7 @@ executable dotenv
 
 library
   exposed-modules:      Configuration.Dotenv
+                      , Configuration.Dotenv.Environment
```

Comment:
@habibalamin Is it necessary to expose this module?

Reply:
It’s not necessary, but I thought a module which wraps the CPP stuff we have to do would be generally useful for many other people.

For example, at certain points, they may themselves want to write their own functions which wrap `parseFile`, so they can write their own `loadFile`, just like I do [here](https://github.com/stackbuilders/dotenv-hs/pull/7#issuecomment-280757759) before the blank environment variable issue was fixed.

Of course, if they require blank environment variables, like I do, then they’ll just have to use the `System.Environment.Blank`, but if they don’t care about that, but do want to wrap `parseFile` for other reasons, they’d probably want to use the same environment functions we use (for consistency with unwrapped behaviour by default), which are automatically selected based on version of `base`, instead of writing their own CPP stuff like we did.

Reply:
Ok, I think you have a point @habibalamin .  Let's leave that module exposed.